README.md

```diff
@@ -319,4 +320,4 @@ The logo was made by [alexchoffy](https://www.instagram.com/alexchoffy/).
 
 ## Blog Posts
 
-[Clean Configuration Management in Golang](https://dev.to/ilyakaznacheev/clean-configuration-management-in-golang-1c89).
+[Clean Configuration Management in Golang](https://dev.to/ilyakaznacheev/clean-configuration-management-in-golang-1c89).
```

Comment:
Please return newline at the end of the readme

Reply:
done ✅